core/src/main/java/com/github/steanky/element/core/data/BasicDataInspector.java

```diff
@@ -51,8 +51,7 @@ public BasicDataInspector(final @NotNull KeyParser idParser) {
                 final boolean isIterable;
                 if (TypeUtils.isAssignable(returnType, COLLECTION_TYPE)) {
                     isIterable = true;
-                }
-                else  {
+                } else {
```

Comment:
we have evolved from this

Reply:
isIterable is used after if/else + the else block has an additional statement so i can't just use ternary

Reply:
}
else {

:kclu:

Reply:
i'm not changing the .editorconfig